python/dgl/graphbolt/utils/sample_utils.py

```diff
@@ -8,6 +8,72 @@
 from ..base import etype_str_to_tuple
 
 
+def find_reverse_edges(
```

Comment:
What about `add_reverse_edges`?

Reply:
add_reverse_edges is slightly better than find_reverse_edges. It is pretty hard to find a name :(

Reply:
just changed back from `add_reverse_edges`... If both you think it's better, let me change back